src/Sarif.Viewer.VisualStudio/Services/ILoadSarifLogService.cs

```diff
@@ -15,5 +18,29 @@ internal interface ILoadSarifLogService
         /// The path to the file from which the SARIF log should be loaded.
         /// </param>
         void LoadSarifLog(string path);
+
+        /// <summary>
+        /// Loads the specified SARIF logs in the viewer.
+        /// </summary>
+        /// <param name="paths">The complete path to the SARIF log files.</param>
+        void LoadSarifLogs(IEnumerable<string> paths);
+
+        /// <summary>
+        /// Loads the specified SARIF logs in the viewer.
+        /// </summary>
+        /// <param name="paths">The complete path to the SARIF log files.</param>
+        /// <param name="promptOnLogConversions">Specifies whether the viewer should prompt if a SARIF log needs to be converted.</param>
+        /// <remarks>
+        /// Reasons for SARIF log file conversion include a conversion from a tool's log to SARIF, or a the SARIF schema version is not the latest version.
+        /// </remarks>
+        void LoadSarifLogs(IEnumerable<string> paths, bool promptOnLogConversions);
```

Comment:
>LoadSarifLogs [](start = 13, length = 13)

Can we just collapse these into a single overload and provide a default value for 'promptOnLogConversions'?

Reply:
Sure. FYI that was existing code (moved from the now-deleted `ILoadSarifLogService2`), but I'll take care of it.

---
In reply to: [520858936](https://github.com/microsoft/sarif-visualstudio-extension/pull/265#discussion_r520858936) [](ancestors = 520858936)